experimental/templates/multiplatform-template/README.md

```diff
@@ -1,8 +1,58 @@
-Compose Multiplatform Application
+# Compose Multiplatform Application
 
-**Desktop**
+Сan run on desktop, Android and iOS.
+Note that iOS target is experimental so far.
+
+*Prerequisites*: to run on iOS, you should have "Kotlin Multiplatform Mobile" plugin installed 
+                 in Android Studio with [installed CocoaPods](https://kotlinlang.org/docs/native-cocoapods.html).
+
+## How to run
+
+Choose a run configuration for an appropriate target in Android Studio and run it.
+
+## Configuring iOS application and running it on a real iOS device
```

Comment:
After discussion, we agreed that this section is useful, because it applies only to this template, not to a general multiplatform project.

But to not confuse people, I would write this section this way and move it to the end:

## iOS

To work with the iOS target you need:
- [Xcode](https://developer.apple.com/xcode/) (to setup the environement)
- [Android Studio](https://developer.android.com/studio)
- ["Kotlin Multiplatform Mobile" plugin](https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile) (to work with all supported targets, including iOS)
- [CocoaPods](https://kotlinlang.org/docs/native-cocoapods.html)

Before trying to open the project in Android Studio, make sure that you have a correct environment for iOS development. Try to follow official Apple tutorials how to create a sample project in Xcode and register a simulator there (or a real device, if you have one).

### Running on an iOS simulator

If you have a correct environment, Android Studio will show available iOS devices on which you can run the application.

### Running on a real iOS device

To run on a real iOS device you need:
- [Team ID](https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/#:~:text=A%20Team%20ID%20is%20a,developer%20in%20App%20Store%20Connect.)
- set this Team ID in `iosApp/Configuration/Config.xcconfig` in the TEAM_ID field
- the registered iOS device in Xcode

After that you can open the project in Android Studio, and it will show the registered iOS device in run configurations.

### Configuring the iOS application

This template contains `iosApp/Configuration/Config.xcconfig` that allows you to configure the most basic properties, without using Xcode. It contains:
- `APP_NAME` - target executable and application bundle name
- `BUNDLE_ID` - [bundle identifier](https://developer.apple.com/documentation/appstoreconnectapi/bundle_ids)
- `TEAM_ID` - [Team ID](https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/#:~:text=A%20Team%20ID%20is%20a,developer%20in%20App%20Store%20Connect.)

Note, that to configure the `APP_NAME` setting, you should open `Config.xcconfig` in any text editor *before opening* the project in Android Studio and correct respective value.

If you need to change this setting after you open the project in Android Studio, please do the following:
 - close the project in Android Studio
 - run `./cleanup.sh` script in a terminal
 - correct the setting
 - open the project in Android Studio again

To configure advanced settings you need to use Xcode. Open `iosApp/iosApp.xcworkspace` in Xcode after opening the project in Android Studio.

(I haven't check everything I wrote, just provided an example)

Reply:
applied